### DISCUSSION TITLE
[BUG] Ampliar la zona del diagrama del flujo de una SM + alinear la linea entre SM de un mismo flujo

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#225] [4. Bug Reports] [answered]
Hola

Me gustaría que si pueden ampliar la zona del diagrama de flujo, porque se ve muy pequeño y al usar el ctrl + scroll, a veces aumenta la vista del navegador (Punto 1)

Con respecto al punto 2 se ve desalineado la división entre maquinas de estados, cómo si es pegado aparte.

Adjunto imagenes de los puntos 1 y 2
![image](https://user-images.githubusercontent.com/61419471/120521240-ba246e00-c3a2-11eb-8caf-bf1d88894845.png)

Gracias :>

**Comment by gmoplaza** [comment, marked as answer]
@NicolasPLG no logro reproducir el problem que estás mostrando. Quizás es por el navegador que estás usando. Te sugiero usar chrome.

**Reply by NicolasPLG** [reply]
hola @gmoplaza el SS corresponde al navegador chrome, te mando interno las credenciales

**Reply by gmoplaza** [reply]
Efectivamente es un bug visual producto del largo de los textos, lo voy a dejar reportado.

**Reply by NicolasPLG** [reply]
gracias @gmoplaza cierro el tema
